src/daml/Daml/Trading/Reward/Badge.daml

```diff
@@ -1 +1,45 @@
 module Daml.Trading.Reward.Badge where 
+
+import Daml.Trading.Account.Account qualified as CreateAccount
+
+template ClaimBadgeRequest
```

Comment:
we should create a choice inside Account to redeem points for badges instead of creating a Request template

Reply:
already fixed it